gpioioctl/Testing.md

```diff
@@ -0,0 +1,73 @@
+# Testing
```

Comment:
This should go in https://github.com/periph/cmd/tree/main/periph-smoketest/gpiosmoketest

Reply:
I've done the restructure. The sysfs stuff is still conflicting. Do you understand it?